tests/test_models/nn/test_deepar.py

```diff
@@ -129,13 +130,20 @@ def test_prediction_interval_run_infuture(example_tsds):
     model.fit(example_tsds)
     future = example_tsds.make_future(future_steps=horizon, tail_steps=horizon)
     forecast = model.forecast(ts=future, prediction_size=horizon, prediction_interval=True, quantiles=[0.025, 0.975])
+
+    assert forecast.prediction_intervals_names == ("target_0.025", "target_0.975")
+    prediction_intervals = forecast.get_prediction_intervals()
     for segment in forecast.segments:
         segment_slice = forecast[:, segment, :][segment]
         assert {"target_0.025", "target_0.975", "target"}.issubset(segment_slice.columns)
         assert (segment_slice["target_0.975"] - segment_slice["target_0.025"] >= 0).all()
         assert (segment_slice["target"] - segment_slice["target_0.025"] >= 0).all()
         assert (segment_slice["target_0.975"] - segment_slice["target"] >= 0).all()
 
+        segment_intervals = prediction_intervals[segment]
```

Comment:
Can you explain what are we testing here?

Reply:
In this code fragment, we are testing that correct intervals returned from `get_prediction_intervals`